metadata.json

```diff
@@ -27,18 +27,26 @@
     }
   ],
   "operatingsystem_support": [
+    {
+      "operatingsystem": "Ubuntu",
+      "operatingsystemrelease": [
+        "16.04"
```

Comment:
did you check if 18.04 works as well?

Reply:
debian 8 added also. ( though testing it locally destroyed my laptop :-( )